integration_test/cases/client_test.exs

```diff
@@ -78,7 +78,6 @@ defmodule ClientTest do
       {:disconnect, _},
       {:connect, _},
       {:handle_status, _},
-      {:handle_status, _},
```

Comment:
Since we disconnect, the lose the ets table and the stage, so we never get to call handle_status/2 the second time around.